README.md

```diff
@@ -292,7 +292,10 @@ You can use all of the following options with standalone version on <redoc> tag
   * `links`
     * `color`: # COMPUTED: colors.primary.main
     * `visited`: # COMPUTED: typography.links.color
-    * `hover`: # COMPUTED: lighten(0.2 typography.links.color)
+    * `hover`:
```

Comment:
I would not make a breaking change, instead I would add `hoverTextDecoration` and `textDecoration`